src/Context/AuthContext/AuthContext.js

```diff
@@ -1,16 +1,19 @@
+import axios from "axios";
 import { useEffect, createContext, useContext, useReducer } from "react";
 import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+ 
 
 const AuthContext = createContext()
 
 const AuthContextProvider = ({children}) =>{
+    
+    const getToken = localStorage.getItem("login-Token")
```

Comment:
As you are using this "Token" string in multiple places, you can create a constant for this and use it so that by mistake the value is not edited.